org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/LogListener.java

```diff
@@ -29,4 +29,10 @@ public interface LogListener {
 
   void debug(@Nullable String msg, boolean fromAnalyzer);
 
+  /**
+   *  This should only be used for IDE-specific logging and is not intended for tracing messages from SLCORE as these
+   *  ones are handled like debug messages. IDE-specific logging is something like adaptations, interaction with an
+   *  extension point, ...
+   */
+  void trace(@Nullable String msg);
```

Comment:
I would suggest to rename this method to something more descriptive like `traceIdeMessage`, otherwise it is guaranteed to be confused with SLCORE trace method, even though you have nice comments on top :)

Reply:
Done